src/OpenTelemetry.Exporter.OpenTelemetryProtocol/.publicApi/Stable/PublicAPI.Shipped.txt

```diff
@@ -32,7 +32,6 @@ override OpenTelemetry.Exporter.OtlpLogExporter.Export(in OpenTelemetry.Batch<Op
 override OpenTelemetry.Exporter.OtlpMetricExporter.Export(in OpenTelemetry.Batch<OpenTelemetry.Metrics.Metric!> metrics) -> OpenTelemetry.ExportResult
 override OpenTelemetry.Exporter.OtlpMetricExporter.OnShutdown(int timeoutMilliseconds) -> bool
 override OpenTelemetry.Exporter.OtlpTraceExporter.Export(in OpenTelemetry.Batch<System.Diagnostics.Activity!> activityBatch) -> OpenTelemetry.ExportResult
-override OpenTelemetry.Exporter.OtlpTraceExporter.OnShutdown(int timeoutMilliseconds) -> bool
```

Comment:
I'm not sure how this entry was added earlier; it had a protected access modifier.